cf_xarray/accessor.py

```diff
@@ -1346,6 +1346,19 @@ def rename_like(
             theirs = _single(_get_all)(other, key)[0]
             renamer[ours] = theirs
```

Comment:
Are there weird cases where this is actually a conflict and we should skip and warn about it?

Reply:
no idea ;) lets find out.